tests/unit/HelperTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use Mockery as m;
+use Illuminate\Container\Container;
+
+class HelperTest extends PHPUnit_Framework_TestCase
+{
+	public function setUp()
+	{
+		parent::setUp();
+
+		$store = m::mock(anlutro\LaravelSettings\SettingStore::class);
+
+		Container::getInstance()->bind('setting', function() use ($store) {
```

Comment:
`Container::getInstance()` doesn't exist in Laravel 4.

Reply:
Not sure how to deal with the L4 stuff, any tip?

Reply:
To be honest most Laravel 4 code bases are just B/C and probably not looking in using new features so the follow (untested) should be fine:

```
if (version_compare(\Illuminate\Foundation\Application::VERSION, '5.0', '<')) {
    $this->markTestSkipped('Laravel 5 feature.');
    return;
}
```

Reply:
I thought about that but Foundation is not in the composer.json file. Should it be enough by just checking class existance?